packages/xo-common/src/api-errors.js

```diff
@@ -197,3 +197,16 @@ export const alteredAuditRecord = create(23, ({ id, record, nValid }) => ({
   },
   message: 'altered record',
 }))
+
+export const notEnoughResources = create(
+  24,
+  ({ resourceSet, resourceType, available, requested }) => ({
+    data: {
+      resourceSet,
+      resourceType,
+      available,
+      requested,
```

Comment:
Maybe it should be an array, in case one day we check multiple resources at once?

Reply:
I always say that we should avoid answering hypothetical needs, but in this case, the change would be minor, but if we need this someday, we won't have to create a new, separate error type.